src/Locale/ta/Entities.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Facilities": "Facilities",
+  "Patients": "Patients",
+  "Sample Test": "Sample",
+  "Shifting": "Shifting",
+  "Resource": "Resource",
+  "External Results": "External",
+  "Users": "Users",
+  "Profile": "Profile",
+  "Dashboard": "Dashboard"
+}
```

Comment:
@shrihari689 It would be great if you translate these 😄

Reply:
Wohooooo 🥳 Done @bodhish 🚀